variables.tf

```diff
@@ -243,6 +243,12 @@ variable "rds_deletion_protection" {
   default     = false
 }
 
+variable "rds_version" {
+  type        = string
+  description = "The DB version to use for the RDS instance"
+  default     = "11.8"
```

Comment:
i would put the default to the latest version of rds no?

Reply:
Good point, kept 11.8 by default not to break anyone's setup, but they can customize it to 11.8 if they prefer. Do we have tests that verify successful RDS upgrades?

Reply:
hmm no, i dont think so